JOINT_NAMES = ['arm1_Joint', 'arm2_Joint', 'arm3_Joint', 'arm4_Joiint', 'arm5_Joint']
NAMED_POSES = {
    "up":     [0.0, 0.0, 0.0, 0.0, 0.0],
    "down":   [0.0, -1.5708, 0.0, 0.0, 0.0],
    "left":   [-1.5708, -1.5708, 0.0, 0.0, 0.0],
    "right":  [1.5708, -1.5708, 0.0, 0.0, 0.0],
    "crouch": [0.0, 1.5708, -1.5708, -1.5708, 0.0]
}
GROUP_NAME = "arm_group"
